src/filters/ws.rs

```diff
@@ -241,8 +241,7 @@ impl Stream for WebSocket {
                 Err(::tungstenite::Error::Io(ref err)) if err.kind() == WouldBlock => {
                     return Ok(Async::NotReady);
                 }
-                Err(::tungstenite::Error::ConnectionClosed(frame)) => {
-                    trace!("websocket closed: {:?}", frame);
+                Err(::tungstenite::Error::ConnectionClosed) => {
                     return Ok(Async::Ready(None));
```

Comment:
Shouldn't the `trace!("websocket closed: {:?}", frame);` be logged here?

The websocket is closed when we receive `ConnectionClosed`, when the server receivs `Message::Close` it does not automatically mean that the websocket is closed, but rather that we received a "request" to close the websocket connection, i.e. it's like a message from the client: "Hey Server, I want to close the connection, please confirm and drop the TCP connection".